recipe/meta.yaml

```diff
@@ -10,25 +10,24 @@ source:
   sha256: 4b7bfbd7d34a11673d80dfae7fabbf921c139aac78383fb0ca16a90d3b53a66e
 
 build:
-  noarch: python
+  skip: true  # [py<37]
   script: 
-    - {{ PYTHON }} -m pdm build -vv
-    - {{ PYTHON }} -m pip install dist/unearth-{{ version }}-py3-none-any.whl -vv
+    - python -m pip install --no-deps --ignore-installed .
   entry_points:
     - unearth = unearth.__main__:cli
   number: 0
 
 requirements:
   host:
     - pip <22.2
-    - python >=3.7
-    - pdm
+    - python
     - pdm-pep517
     - setuptools
+    - wheel
   run:
     - cached-property >=1.5.2
```

Comment:
Only needed for python 3.7. and older: https://github.com/frostming/unearth/blob/0.6.1/pyproject.toml#L17